packages/integrations/vercel/README.md

```diff
@@ -172,6 +172,30 @@ import astroLogo from '../assets/logo.png';
 />
 ```
 
+### devImageService
+
+**Type:** `'sharp' | 'squoosh' | string`<br>
+**Available for:** Serverless, Static
+**Added in:** `@astrojs/vercel@3.3.0`
+**Default**: 'sharp'
+
+Allows you to configure a which image service to use in development. This can be useful if you cannot install Sharp's dependencies on your development machine and prefer to use Squoosh instead.
```

Comment:
```suggestion
Allows you to configure which image service to use in development. This can be useful if you cannot install Sharp's dependencies on your development machine, but using another image service like Squoosh would allow you to preview images in your dev environment.
```
I know this might sound like a little much, but I wonder whether adding a purpose to this statement (pick the reason you think makes sense!) helps to describe the choice and also really emphasize that this is JUST for development. (Build is unaffected.  <-- and you could even add something like that, if you wanted!)